Jenkinsfile

```diff
@@ -178,7 +162,7 @@ node {
 String bumpMinorVersion(String version) {
     def parts = version.tokenize('.')
     if (parts.size() != 3) {
-        throw new IllegalArgumentException("${version} is not a valid MAJOR.MINOR.PATCH version")
+        error "${version} is not a valid MAJOR.MINOR.PATCH version"
```

Comment:
The previous code never worked in Jenkins because you can't `new` things in this script due to security restrictions.